app/views/admin/facility_groups/_form.html.erb

```diff
@@ -26,19 +26,14 @@
                         disabled: action_name == "edit" ? true : false %>
       <% end %>
 
-      <% if facility_group.facilities.present? %>
-        <%= form.collection_check_boxes :facility_ids, facility_group.facilities, :id, :name, label: "Associated facilities", checked_value: 1 %>
```

Comment:
This deletion means we should be able to delete something in the corresponding controller too, right?